h2o-py/tests/testdir_generic_model/pyunit_mojo_ids.py

```diff
@@ -40,11 +40,11 @@ def test_mojo_ids():
     # Test initialize model_id from path
```

Comment:
# Test do not initialize model_id from path

Reply:
got it thanks, will fix if there is enough time but since it goes only to rel-zizler and just to tests - I might as well leave as is